File: lista_de_exercicios/questao1/questao1.go
package main
import "fmt"

type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}

func (c Carro) ExibirDetalhes() string {
    return fmt.Sprintf("Marca: %s, Modelo: %s, Ano: %d", c.Marca, c.Modelo, c.Ano)
}

func main() {
    carro1 := Carro{"Toyota", "Corolla", 2020}
    carro2 := Carro{"Ford", "Mustang", 2021}
    carro3 := Carro{"Honda", "Civic", 2022}

    fmt.Println(carro1.ExibirDetalhes())
    fmt.Println(carro2.ExibirDetalhes())
    fmt.Println(carro3.ExibirDetalhes())
}

File: lista_de_exercicios/questao1/questao1.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        return f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}"

carro1 = Carro("Toyota", "Corolla", 2020)
carro2 = Carro("Ford", "Mustang", 2021)
carro3 = Carro("Honda", "Civic", 2022)

print(carro1.exibir_detalhes())
print(carro2.exibir_detalhes())
print(carro3.exibir_detalhes())
